docs/_layouts/rule.html

```diff
@@ -39,3 +40,25 @@ <h5>Schema</h5>
 <pre>
 {{page.optionsJSON}}
 </pre>
+
+{% if page.codeExamples %}
+<div class="wrapper__code-examples">
+    <h3>Code examples: </h3>
+    {% for codeExample in page.codeExamples %}
+    <div class="wrapper__code-example">
+        <h4>{{ codeExample.description }}</h4>
+        {{ codeExample.config | markdownify }}
+        <h5>Passes</h5>
+        <div class="code-example pass">
```

Comment:
one class naming rule I like is for "decorating" class names, in this case `pass` and below `fail`, is to preface them with `-` (e.g. `-pass`, `-fail`) so that css rules for `pass` or `fail` wouldn't apply. By prefacing decorator class names with dashes, you know that it wouldn't be affected by other styles since all other styles for prefaced class names would be scoped to a particular class name.